ext/opentelemetry-exporter-cloud-monitoring/tests/test_cloud_monitoring.py

```diff
@@ -270,6 +278,8 @@ def test_export(self):
         point.value.int64_value = 1
         point.interval.end_time.seconds = WRITE_INTERVAL + 1
         point.interval.end_time.nanos = 0
+        point.interval.start_time.seconds = 1
```

Comment:
Should verify `point.interval.start_time.nanos` is correct too

Reply:
done